ToolsTest/Models/VladyslavClassArticleTests/UpdateTextMethod_MSTest.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Tools.Models;
+
+namespace ToolsTest.Models.VladyslavClassArticleTests
+{
+    [TestClass]
+    public class UpdateTextMethod_MSTest
+    {
+        [TestMethod]
+        public void UpdateText_Text_NotNull()
+        {
+            //Arrange
+            Article _article = new Article();
+            string _text = "Hello";
+            //Act
+            _article.UpdateText(_text);
+            //Assert
+            Assert.IsNotNull(_article);
+        }
+        [TestMethod]
+        public void UpdateText_TextAndUser_NotNull()
+        {
+            //Arange 
+            Article _article = new Article();
+            _article.Text = "Test DONE";
+            _article.Editor = new User("John");
+            //Act
+            _article.UpdateText(_article.Text, _article.Editor);
```

Comment:
Here the `Text` and `Editor` properties are changed directly, so test does not quaranties that the changes are happen exactly due to `UpdateText` method.
```suggestion
            string text = "Test DONE";
            User editor = new User("John");
            //Act
            _article.UpdateText(text, editor);
```